### PR TITLE
Drone: Enable master branch testing

### PR DESCRIPTION
**Pull request by aknuds1**
**What this PR does / why we need it**:
Enable testing of master branch in Drone.

Publishing steps are included, but only simulated until we disable CircleCI's master pipeline.

**NB:** .drone.yml is auto-generated from .drone.star, but you might want to read it just to verify the Drone configuration.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -233,5 +225,282 @@ services:
...
+  - test-frontend
+
+- name: build-windows-installer
+  image: grafana/wix-toolset-ci:v3
+  commands:
+  - echo ./scripts/build/ci-msi-build/ci-msi-build-oss.sh
+  depends_on:
+  - package
+
+- name: release-next-npm-packages
```
Comment (marefr) [inline .drone.yml:460]: This is already done by CircleCI master build, correct? If so, need to be removed for now.

File: .drone.yml [hunk 2]
```diff
@@ -233,5 +225,282 @@ services:
...
+  - npx lerna bootstrap
+  - echo //registry.npmjs.org/:_authToken=$${NPM_TOKEN} >> ~/.npmrc"
+  - echo ./scripts/circle-release-next-packages.sh
+  environment:
+    NPM_TOKEN:
+      from_secret: npm_token
+  depends_on:
+  - end-to-end-tests
+
+- name: publish-packages
```
Comment (marefr) [inline .drone.yml:472]: This is already done by CircleCI master build, correct? If so, need to be removed for now.

**Review by marefr — CHANGES_REQUESTED** [review]
I was going to approve, but seems like it will run publish steps that's already in CircleCI, see comments.

File: .drone.yml (hunk 1 above)
Comment (aknuds1) [reply .drone.yml:460]: @marefr No need to worry, the publishing is only simulated (see `echo` at line 465).

File: .drone.yml (hunk 2 above)
Comment (aknuds1) [reply .drone.yml:472]: @marefr No need to worry, the publishing is only simulated (see `echo` at line 475).

**Review by marefr — APPROVED** [review]
Oops :+1: LGTM

**Pull request merged**